app/code/community/Bolt/Boltpay/Model/Order.php

```diff
@@ -263,12 +272,20 @@ public function createOrder($reference, $sessionQuoteId = null, $isPreAuthCreati
             $this->associateOrderToCustomerWhenPlacingOnPDP($order->getData('increment_id'));
         }
 
+        ///////////////////////////////////////////////////////
+        /// Dispatch order save events
+        ///////////////////////////////////////////////////////
         $recurringPaymentProfiles = $service->getRecurringPaymentProfiles();
-
+        
         Mage::dispatchEvent(
             'checkout_submit_all_after',
             array('order' => $order, 'quote' => $immutableQuote, 'recurring_profiles' => $recurringPaymentProfiles)
         );
+
+        Mage::dispatchEvent(
+            'bolt_boltpay_order_creation_after',
```

Comment:
Do we need this event given merchant can just use `checkout_submit_all_after` and check payment method?